python/housinginsights/ingestion/LoadData.py

```diff
@@ -286,8 +289,267 @@ def rebuild(self):
 
             processed_data_ids.append(manifest_row['unique_data_id'])
 
+        # build Zone Facts table
+        self._create_zone_facts_table()
+
         return processed_data_ids
 
+    def _create_zone_facts_table(self):
+        """
+        Creates the zone_facts table which is used as a master table for API
+        endpoints. The purpose is to avoid recalculating fields adhoc and
+        performing client-side reconfiguration of data into display fields.
+        This is all now done in the backend whenever new data is loaded.
+
+        :return: a immutable dict that contains the tables structure of the
+        table and sqlAlchemy metadata object used.
+        """
+        # drop zone_facts table if already in db
+        if 'zone_facts' in self.engine.table_names():
+            with self.engine.connect() as conn:
+                conn.execute('DROP TABLE zone_facts;')
+
+        # create empty zone_facts table
+        metadata = MetaData(bind=self.engine)
+        zone_facts = Table('zone_facts', metadata,
+                           Column('zone', String, primary_key=True),
```

Comment:
Should add a column here for 'zone_type' which has 'ward', 'neighborhood_cluster', or 'census_tract' as the field value. This is for convience when getting the data out of the table later - instead of a sql statement with a `where in (big,long,list,of,zones)` we can just use `where zone_type='ward'`.